run-platform.sh

```diff
@@ -268,9 +268,26 @@ build_services() {
   fi
 }
 
+prepare_for_first_setup() {
+  pushd ${script_dir}/docker 1>/dev/null
+
+  echo -e "$blue_text""Preparing for initial setup""$default_text"
+  VERSION=$opt_version $docker_compose_cmd up -d db redis
+
+  echo -e "Creating a schema for Unstract in the database"
+  VERSION=$opt_version $docker_compose_cmd run backend .venv/bin/python manage.py create_schema
```

Comment:
@chandrasekharan-zipstack schema creation should be part of init script (because it creates schema only if does't exists). so we can  avoid touching the backend Dockerfile and some scripts used by OSS users. It is more elegant way to handle this issue.

Reply:
@chandrasekharan-zipstack Agree here.

Extend entrypoint script itself with necessary Django commands so that we can replace this with `VERSION=$opt_version $docker_compose_cmd run backend prepare_and_migrate` here.

Also we can hopefully remove Redis dependency later, as ideally db migrations do not have anything to do with Redis.

Reply:
Yes @hari-kuriakose, will retain this function and let the schema creation be done through the new CLI argument.
The redis dependency comes from the celery tasks that we register in backend's [__init__.py](https://github.com/Zipstack/unstract/blob/0ee58329340b2c97520308140e5031f033485e9e/backend/backend/__init__.py#L1)

Reply:
@chandrasekharan-zipstack Ah yes, we will need to do this later.